integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -187,6 +188,13 @@ public Long load(String groupName) {
     mIsUserGroupTranslation = conf.getBoolean(PropertyKey.FUSE_USER_GROUP_TRANSLATION_ENABLED);
     mMaxUmountWaitTime = (int) conf.getMs(PropertyKey.FUSE_UMOUNT_TIMEOUT);
     mAuthPolicy = AuthPolicyFactory.create(mFileSystem, conf, this);
+    if (opts.isDebug()) {
+      try {
+        LogUtils.setLogLevel(this.getClass().getCanonicalName(), "DEBUG");
```

Comment:
![image](https://user-images.githubusercontent.com/17329931/150685537-77f70377-7592-4c00-a64a-03f0c038642d.png)

Since `getLogger` use `clazz.getName()` as logger name, I suggest to use `this.getClass().getName()`  here